pkg/git/libgit2/checkout_test.go

```diff
@@ -77,49 +77,29 @@ func TestCheckoutBranch_Checkout(t *testing.T) {
 	}
 
 	tests := []struct {
-		name                   string
-		branch                 string
-		filesCreated           map[string]string
-		lastRevision           string
-		expectedCommit         string
-		expectedConcreteCommit bool
-		expectedErr            string
+		name           string
+		branch         string
+		filesCreated   map[string]string
+		lastRevision   string
```

Comment:
`filesCreated` is not being used any more.
`lastRevision` doesn't apply to this test as unmanaged transport doesn't support no-op clone.